pkg/asset/manifests/dns.go

```diff
@@ -3,6 +3,7 @@ package manifests
 import (
 	"context"
 	"fmt"
+	"github.com/openshift/installer/pkg/types/ovirt"
```

Comment:
This should be moved to the final block of imports.

Reply:
done